docs/hybrid-slurm-cluster/blueprints/create-networks.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: peering-networks
+
+vars:
+  project_id: <PROJECT_A_ID>  # The project ID for the static cluster
+  project_id_compute: <PROJECT_B_ID>  # The project ID for the burst compute VMs
+  deployment_name: peering-networks-demo
+  region: us-central1
+  zone: us-central1-c
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded module, denoted by "modules/*" without ./, ../, /
+  # as a prefix. To refer to a local or community module, prefix with ./, ../ or /
+  # Example - ./modules/network/vpc
+  - source: modules/network/vpc
+    kind: terraform
+    id: network0
+    settings:
+      network_name: static-cluster-network
+      subnetwork_name: primary-subnet
+      network_address_range: 10.0.0.0/16
+      subnetwork_size: 0
```

Comment:
Note for self: This parameter is deprecated (https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/network/vpc#input_subnetwork_size).
It was added because the default value was added to the bits in my cidr range causing other problems, try using [default_primary_subnetwork_size](https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/network/vpc#input_default_primary_subnetwork_size)